ontohgis.ttl

```diff
@@ -17236,6 +16854,21 @@ ontohgis:administrative_system_48 rdf:type owl:NamedIndividual ,
                                                                                  "Słowacja (1996-2020)"@pl .
 
 
+###  http://purl.org/ontohgis#administrative_system_49
+ontohgis:administrative_system_49 rdf:type owl:NamedIndividual ,
+                                           ontohgis:object_21 ;
+                                  ontohgis:endsDominationAt "1806-08-06"^^xsd:date ;
+                                  ontohgis:hasBeginning "800-12-25"^^xsd:date ;
```

Comment:
Pellet reasoner throws the following error:
An error occurred during reasoning: Cannot do reasoning with inconsistent ontologies!
Reason for inconsistency: Literal value "800-12-25"^^date is not valid for the rdatatype date.
org.mindswap.pellet.exceptions.InconsistentOntologyException: Cannot do reasoning with inconsistent ontologies!
Reason for inconsistency: Literal value "800-12-25"^^date is not valid for the rdatatype date

I think the value should be "0800-12-25"

Reply:
ok